eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -152,11 +147,11 @@ instance Group (MultiAsset c) where
   invert (MultiAsset m) =
     MultiAsset (canonicalMap (canonicalMap ((-1 :: Integer) *)) m)
 
-instance CC.Crypto c => DecodeMint (MultiAsset c) where
-  decodeMint = decodeMultiAssetMaps decodeIntegerBounded64
+instance CC.Crypto c => FromCBOR (MultiAsset c) where
+  fromCBOR = decodeMultiAssetMaps decodeIntegerBounded64
 
-instance CC.Crypto c => EncodeMint (MultiAsset c) where
-  encodeMint = encodeMultiAssetMaps
+instance CC.Crypto c => ToCBOR (MultiAsset c) where
+  toCBOR = encodeMultiAssetMaps
```

Comment:
Is this `encodeMultiAssetMaps` function even necessary? It seems to be doing only `toCBOR m`.  But maybe for symmetry with `decodeMultiAssetMaps`

Reply:
Yeah, I think it is good to keep it for consistency with `decodeMultiAssetMaps`. Both of these are internal functions, so there is no worry of anyone using it outside this module.